package.json

```diff
@@ -78,7 +79,8 @@
 			"@typescript-eslint/restrict-plus-operands": "off"
 		},
 		"globals": [
-			"browser"
+			"browser",
+			"chrome"
```

Comment:
No. We don’t use the chrome variable directly so it should stay blocked.

Reply:
I removed this change.